ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -242,6 +241,7 @@ object KeyValueConsumption {
       txId: DamlLogEntryId,
       tx: SubmittedTransaction): CommittedTransaction =
     /* Assign absolute contract ids */
+    /* Assign absolute contract ids */
```

Comment:
Duplicate comment line.

Reply:
You are right it is not store in the transaction. :-S 

This is work in progress and this seed should be store somewhere by the ledger implementation.